File: ServerSpace/src/main/java/Business/ServLista.java
package Business;

import EntityLayer.ServListaEntity;
import java.util.ArrayList;
import Framework.Inj;
import DataLayer.ServListaDB;

public class ServLista {

    public ArrayList<ServListaEntity> ObtenerCodigo(String codigo) {
        ServListaDB BD = new ServListaDB();
        return BD.ObtenerCodigo(codigo);
    }

    public ArrayList<ServListaEntity> Buscar(String codigo, String Nombre) {
        ServListaDB BD = new ServListaDB();
        return BD.Buscar(codigo, Nombre);
    }

    public ArrayList<ServListaEntity> ObtenerMain(String codigo) {
        ServListaDB BD = new ServListaDB();
        return BD.ObtenerMain(codigo);
    }

    public ServListaEntity Registrar(ServListaEntity Item) {
        try {
            Inj.IniciarTranssaccion(true);
            ServListaDB BD = new ServListaDB();
            BD.Registrar(Item);
            Inj.FinalizarTranssaccion();
            return Item;
        } catch (Exception ex) {
            System.out.println("ERROR " + ex);
            throw new UnsupportedOperationException("MENSAJE :" + ex);
        }
    }

    public ArrayList<ServListaEntity> ObtenerItem(int ListaId) {
        ServListaDB BD = new ServListaDB();
        return BD.ObtenerItem(ListaId);
    }

}

File: SharedApi/Space/Space.BusinessLayer/ServLista.cs
using Framework;
using Space.DataLayer;
using Space.EntityLayer;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Space.BusinessLayer
{
    public class ServLista
    {
        public static List<ServListaEntity> ObtenerServListaCodigo(String codigo)
        {
            ServListaDB DB = new ServListaDB();
            return DB.ObtenerServListaCodigo(codigo);
        }
        public static List<ServListaEntity> BuscarServLista(String codigo, String Nombre)
        {
            ServListaDB DB = new ServListaDB();
            return DB.BuscarServLista(codigo, Nombre);
        }

        public static List<ServListaEntity> ObtenerServListaMain(String codigo)
        {
            ServListaDB DB = new ServListaDB();
            return DB.ObtenerServListaMain(codigo);
        }
        public static Int32 Registrar(ServListaEntity Ent)
        {
            ServListaDB DB = new ServListaDB();
            DB.Registrar(Ent);
            return Ent.ListaId;
        }

    
        public static List<ServListaEntity> ObtenerServListaItem(Int32 ListaId)
        {
            ServListaDB DB = new ServListaDB();
            return DB.ObtenerServListaItem(ListaId);
        }
    }
}
